style.css.map

```diff
@@ -1,6 +1,6 @@
C,aACc;E/CzbV,WAAQ,EAAE,OAAsB;;A+C2bpC,aACc;E/C5bV,YAAQ,EAAE,OAAsB;;A+C+bpC,eACgB;E/ChcZ,WAAQ,EAAE,QAAsB;;A+CkcpC,eACgB;E/CncZ,YAAQ,EAAE,QAAsB;;A+CucpC,aACc;E/CxcV,WAAQ,EAAE,OAAsB;EAAhC,YAAQ,EAAE,OAAsB;;A+C0cpC,aACc;E/C3cV,YAAQ,EAAE,OAAsB;EAAhC,aAAQ,EAAE,OAAsB;;A+C8cpC,eACgB;E/C/cZ,WAAQ,EAAE,QAAsB;EAAhC,YAAQ,EAAE,QAAsB;;A+CidpC,eACgB;E/CldZ,YAAQ,EAAE,QAAsB;EAAhC,aAAQ,EAAE,QAAsB;;A+CudpC,aACc;E/CxdV,UAAQ,EAAE,OAAsB;EAAhC,aAAQ,EAAE,OAAsB;;A+C0dpC,aACc;E/C3dV,WAAQ,EAAE,OAAsB;EAAhC,cAAQ,EAAE,OAAsB;;A+C8dpC,eACgB;E/C/dZ,UAAQ,EAAE,QAAsB;EAAhC,aAAQ,EAAE,QAAsB;;A+CiepC,eACgB;E/CleZ,WAAQ,EAAE,QAAsB;EAAhC,cAAQ,EAAE,QAAsB;;;;;A+C6epC,YACa;E/C9eT,MAAQ,EAAE,OAAsB;;A+CgfpC,YACa;E/CjfT,OAAQ,EAAE,OAAsB;;A+CofpC,cACe;E/CrfX,MAAQ,EAAE,QAAsB;;A+CufpC,cACe;E/CxfX,OAAQ,EAAE,QAAsB;;A+C4fpC,aACc;E/C7fV,UAAQ,EAAE,OAAsB;;A+C+fpC,aACc;E/ChgBV,WAAQ,EAAE,OAAsB;;A+CmgBpC,eACgB;E/CpgBZ,UAAQ,EAAE,QAAsB;;A+CsgBpC,eACgB;E/CvgBZ,WAAQ,EAAE,QAAsB;;A+C2gBpC,aACc;E/C5gBV,YAAQ,EAAE,OAAsB;;A+C8gBpC,aACc;E/C/gBV,aAAQ,EAAE,OAAsB;;A+CkhBpC,eACgB;E/CnhBZ,YAAQ,EAAE,QAAsB;;A+CqhBpC,eACgB;E/CthBZ,aAAQ,EAAE,QAAsB;;A+C0hBpC,aACc;E/C3hBV,aAAQ,EAAE,OAAsB;;A+C6hBpC,aACc;E/C9hBV,cAAQ,EAAE,OAAsB;;A+CiiBpC,eACgB;E/CliBZ,aAAQ,EAAE,QAAsB;;A+CoiBpC,eACgB;E/CriBZ,cAAQ,EAAE,QAAsB;;A+CyiBpC,aACc;E/C1iBV,WAAQ,EAAE,OAAsB;;A+C4iBpC,aACc;E/C7iBV,YAAQ,EAAE,OAAsB;;A+CgjBpC,eACgB;E/CjjBZ,WAAQ,EAAE,QAAsB;;A+CmjBpC,eACgB;E/CpjBZ,YAAQ,EAAE,QAAsB;;A+CwjBpC,aACc;E/CzjBV,WAAQ,EAAE,OAAsB;EAAhC,YAAQ,EAAE,OAAsB;;A+C2jBpC,aACc;E/C5jBV,YAAQ,EAAE,OAAsB;EAAhC,aAAQ,EAAE,OAAsB;;A+C+jBpC,eACgB;E/ChkBZ,WAAQ,EAAE,QAAsB;EAAhC,YAAQ,EAAE,QAAsB;;A+CkkBpC,eACgB;E/CnkBZ,YAAQ,EAAE,QAAsB;EAAhC,aAAQ,EAAE,QAAsB;;A+CwkBpC,aACc;E/CzkBV,UAAQ,EAAE,OAAsB;EAAhC,aAAQ,EAAE,OAAsB;;A+C2kBpC,aACc;E/C5kBV,WAAQ,EAAE,OAAsB;EAAhC,cAAQ,EAAE,OAAsB;;A+C+kBpC,eACgB;E/ChlBZ,UAAQ,EAAE,QAAsB;EAAhC,aAAQ,EAAE,QAAsB;;A+CklBpC,eACgB;E/CnlBZ,WAAQ,EAAE,QAAsB;EAAhC,cAAQ,EAAE,QAAsB;;;;;A+C8lBpC,WACY;E/C/lBR,MAAQ,EAAE,IAAsB;;A+CimBpC,WACY;E/ClmBR,OAAQ,EAAE,IAAsB;;A+CqmBpC,aACc;E/CtmBV,MAAQ,EAAE,KAAsB;;A+CwmBpC,aACc;E/CzmBV,OAAQ,EAAE,KAAsB;;A+C6mBpC,YACa;E/C9mBT,UAAQ,EAAE,IAAsB;;A+CgnBpC,YACa;E/CjnBT,WAAQ,EAAE,IAAsB;;A+ConBpC,cACe;E/CrnBX,UAAQ,EAAE,KAAsB;;A+CunBpC,cACe;E/CxnBX,WAAQ,EAAE,KAAsB;;A+C4nBpC,YACa;E/C7nBT,YAAQ,EAAE,IAAsB;;A+C+nBpC,YACa;E/ChoBT,aAAQ,EAAE,IAAsB;;A+CmoBpC,cACe;E/CpoBX,YAAQ,EAAE,KAAsB;;A+CsoBpC,cACe;E/CvoBX,aAAQ,EAAE,KAAsB;;A+C2oBpC,YACa;E/C5oBT,aAAQ,EAAE,IAAsB;;A+C8oBpC,YACa;E/C/oBT,cAAQ,EAAE,IAAsB;;A+CkpBpC,cACe;E/CnpBX,aAAQ,EAAE,KAAsB;;A+CqpBpC,cACe;E/CtpBX,cAAQ,EAAE,KAAsB;;A+C0pBpC,YACa;E/C3pBT,WAAQ,EAAE,IAAsB;;A+C6pBpC,YACa;E/C9pBT,YAAQ,EAAE,IAAsB;;A+CiqBpC,cACe;E/ClqBX,WAAQ,EAAE,KAAsB;;A+CoqBpC,cACe;E/CrqBX,YAAQ,EAAE,KAAsB;;A+CyqBpC,YACa;E/C1qBT,WAAQ,EAAE,IAAsB;EAAhC,YAAQ,EAAE,IAAsB;;A+C4qBpC,YACa;E/C7qBT,YAAQ,EAAE,IAAsB;EAAhC,aAAQ,EAAE,IAAsB;;A+CgrBpC,cACe;E/CjrBX,WAAQ,EAAE,KAAsB;EAAhC,YAAQ,EAAE,KAAsB;;A+CmrBpC,cACe;E/CprBX,YAAQ,EAAE,KAAsB;EAAhC,aAAQ,EAAE,KAAsB;;A+CwrBpC,YACa;E/CzrBT,UAAQ,EAAE,IAAsB;EAAhC,aAAQ,EAAE,IAAsB;;A+C2rBpC,YACa;E/C5rB
```

Comment:
Just a map file that maps the `.scss` to the compiled `.css` when inspecting in a browser, it's checked in for the same reason `style.css` is.